### PR TITLE
New package: Everything.Cli version 1.1.0.26

### PR DESCRIPTION
**Pull request by 1208nn**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

Everything search cli portable official.
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/127299)

### CODE REVIEW

**Comment by 1208nn** [comment]
@microsoft-github-policy-service agree

**Comment by stephengillie** [comment]
Manual Validation ended with:
> PS C:\Users\User\Desktop> C:\Users\User\AppData\Local\Microsoft\WinGet\Packages\voidtools.Everything.Cli__DefaultSource\es.exe Error 8: Everything IPC window not found. Please make sure Everything is running. PS C:\Users\User\Desktop> 
Is `voidtools.Everything` a dependency?

File: manifests/v/voidtools/Everything/Cli/1.1.0.26/voidtools.Everything.Cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+PackageIdentifier: voidtools.Everything.Cli
+PackageVersion: 1.1.0.26
```
Comment (stephengillie) [inline manifests/v/voidtools/Everything/Cli/1.1.0.26/voidtools.Everything.Cli.installer.yaml:2]:
```suggestion
PackageVersion: 1.1.0.26
Dependencies:
  PackageDependencies:
    - PackageIdentifier: voidtools.Everything
```

**Comment by stephengillie** [comment]
Hi @1208nn,

It appears that `voidtools.Everything` is a dependency for this package.

**Comment by 1208nn** [comment]
Hi @stephengillie 

> It appears that `voidtools.Everything` is a dependency for this package.

Any versions of `Everything` installed could make this work, including `Lite`, `Alpha` and even portable.

I think considering `voidtools.Everything` as a dependency is not friendly for these users.

**Comment by stephengillie** [comment]
Hi @1208nn,

The difficulty is for new users, who might not have heard of `voidtools` and do not have any version of `Everything` installed. We're working on a kind of wildcard system for dependencies, which might allow any .NET SDK to be selected, and could be a better solution for this situation. But for now, the path forward isn't clear.

**Comment by 1208nn** [comment]
Hi @stephengillie,

The dependency was added. So is the PR OK to be merged?

**Review by stephengillie — APPROVED** [review]

**Pull request merged**